modules/azurerm_netapp/variables.tf

```diff
@@ -23,6 +23,10 @@ variable "subnet_id" {
   description = "Azure subnet id for Azure NetApp Files"
 }
 
+variable "network_features" {
+  description = "Indicates which network feature to use, accepted values are Basic or Standard, it defaults to Basic if not defined."
+}
```

Comment:
So for this code change we do actually say the default is `Basic` so we need at least that defined here

Reply:
Done.